File: library/network/src/main/java/com/example/network/callback/IInterceptorCallback.java
package com.example.network.callback;

import java.util.List;

import okhttp3.Interceptor;

/**
 * @author fqxyi
 * @date 2018/2/27
 * 拦截器回调接口
 */
public interface IInterceptorCallback {

    List<Interceptor> getInterceptorList();

    List<Interceptor> getNetworkInterceptorList();

}

File: library/network/src/main/java/com/example/network/callback/IInterceptorCallback.kt
package com.example.network.callback

import okhttp3.Interceptor

/**
 * @author fqxyi
 * @date 2018/2/27
 * 拦截器回调接口
 */
interface IInterceptorCallback {

    fun getInterceptorList(): List<Interceptor>?

    fun getNetworkInterceptorList(): List<Interceptor>?
}